string runName
string scenario
---
int64[] timePerAttempt
int64 totalTime
